langchain4j-open-ai/src/main/java/dev/langchain4j/model/openai/OpenAiEmbeddingModel.java

```diff
@@ -32,7 +34,7 @@
  * Represents an OpenAI embedding model, such as text-embedding-ada-002.
  */
 public class OpenAiEmbeddingModel extends DimensionAwareEmbeddingModel implements TokenCountEstimator {
-
+    private static final int REQUEST_TEXT_MAX_ARRAY_SIZE = 2048;
```

Comment:
It would also be great to make it configurable, so that user could override this default in the builder:

```java
OpenAiEmbeddingModel.builder()
    ...
    .maxSegmentsPerBatch(1000)
    .build();
```